jenkins_rock/rockcraft.yaml

```diff
@@ -46,7 +46,7 @@ parts:
       # Use jenkins war rather than apt install for easier Jenkins version control.
       cd ${CRAFT_PART_INSTALL}/srv/jenkins
       curl -sLO https://github.com/jenkinsci/plugin-installation-manager-tool/releases/download/${JENKINS_PLUGIN_MANAGER_VERSION}/jenkins-plugin-manager-${JENKINS_PLUGIN_MANAGER_VERSION}.jar
-      curl -sLO https://mirrors.jenkins.io/war/${JENKINS_VERSION}/jenkins.war
+      curl -sLO https://updates.jenkins.io/download/war/${JENKINS_VERSION}/jenkins.war
```

Comment:
Something that I remember to do today in Synapse rock: add a timeout to curl :) "-m" flag, I guess.